src/els_methods.erl

```diff
@@ -129,7 +129,17 @@ initialize(Params, State) ->
 
 -spec initialized(params(), state()) -> result().
 initialized(_Params, State) ->
-  {noresponse, State#{status => initialized}}.
+  %% Report to the user the server version
+  {ok,     App} = application:get_application(),
+  {ok, Version} = application:get_key(App, vsn),
+  lager:info("initialized: [App=~p] [Version=~p]", [App, Version]),
+  BinVersion = list_to_binary(Version),
+  Message = <<"Erlang LS version: ", BinVersion/binary>>,
+  Method1 = <<"window/showMessage">>,
```

Comment:
Nit: Why `Method1` and not just `Method`? 😸

Reply:
Copy/paste, will clean it up